src/clj_puppetdb/vcr.clj

```diff
@@ -0,0 +1,28 @@
+(ns clj-puppetdb.vcr
+  (:require [clojure.edn :as edn]
+            [me.raynes.fs :as fs]
+            [pandect.algo.sha1 :refer [sha1]]
+            [puppetlabs.http.client.sync :as http]
+            [puppetlabs.kitchensink.core :refer [dissoc-in]]))
+
+(defn- vcr-filename
+  [vcr-dir path]
+  (str vcr-dir "/" (sha1 path) ".clj"))
+
+(defn- vcr-transform
+  [response]
+  "Transform the response to avoid serialization problems"
+  (dissoc-in response [:content-type :charset]))
```

Comment:
I just realized that the requests are always sent with ```:as :text```, but I'm intending to change that when adding support for streamed processing.